styles/lib/carnival/_theme.scss

```diff
@@ -30,3 +30,6 @@ $Grey1: #EEEAE0;
 $Grey2: #757575;
 $black: #000000;
 $white: #FFFFFF;
+
+// Fonts
```

Comment:
make sure to delete this file since it has been replaced by `colors` and `fonts`